### ISSUE TITLE
Create PR view branch background color is quite dark on light theme

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#188648]
![image](https://github.com/microsoft/vscode/assets/900690/61a999fe-7e46-4365-bfc7-80ca6649da22)

Is the intent here to match with the secondary button color? I wonder if that color should be a bit lighter on light theme?

**Comment by hbons** [comment]
This is by design. The style is inherited from secondary buttons as they are technically buttons. Maybe we need a tertiary button style in themes.

**Comment by daviddossett** [comment]
I tend to agree that, even in dark themes, these feel feel like they are closer siblings to inputs then they are to buttons given that they represent a state rather than a one-and-done action.

Have you considered using the input backgrounds and borders @hbons?

**Comment by hbons** [comment]
@daviddossett I did use input colours for a while, but it introduced a lot of edge case bugs where things wouldn't work in all themes (dark/light/hc) at the same time. Hence my suggestion to have special vars for this.

**Comment by hbons** [comment]
I'll have another go at this.
